src/lib/volume_cache.js

```diff
@@ -2,32 +2,13 @@ var debug = require('debug')('docker-worker:volumeCacheManager');
 var path = require('path');
 var Promise = require('promise');
 var fs = require('fs');
-var mkdirp = require('mkdirp');
-var rmrf = require('rimraf');
+const {makeDir, removeDir} = require('util/fs');
```

Comment:
iirc, the correct syntax for importing the local module is `require('./util/fs')`